ui/component/commentsList/view.jsx

```diff
@@ -77,20 +78,21 @@ function CommentList(props: Props) {
     commentsAreExpanded,
     fetchReacts,
     doResolveUris,
+    setCommentListTitle,
     fetchTopLevelComments,
     fetchComment,
     resetComments,
```

Comment:
Seeing this now on File Pages on Desktop.  Probably an indication of something not done right.

<img width="648" alt="image" src="https://user-images.githubusercontent.com/64950861/151311562-4bc955a1-2614-47af-b455-fff7a4aa064e.png">